### ISSUE TITLE
parser: undefined: ignoredTokens

### ISSUE THREAD

**Issue by mewmew** [inspirer/textmapper#10]
I get the compile time error `parser.go:54:2: undefined: ignoredTokens` when trying to compile the Go generated parser for the following grammar ([foo.tm](https://github.com/mewspring/foo/blob/master/foo.tm)):

**Note:** this is on rev 78fc54eedf048929cd4a42ab40b2e1a6160ea31e of Textmapper, using the following command to process the grammar `@java -jar ${TM_DIR}/tm-tool/libs/textmapper-0.9.21.jar foo.tm`

```go
language foo(go);

lang = "foo"
package = "github.com/mewspring/foo"

::lexer

'foobar' : /foobar/

::parser

input : 'foobar' ;
```

**Comment by inspirer** [comment]
Fixed in 76054ba154

**Comment by mewmew** [comment]
> Fixed in 76054ba

Perfect. Thanks!
